types/V1.mli

```diff
@@ -141,7 +141,7 @@ module Flow : sig
 end
 
 (** {1 Connection between endpoints} *)
-module type FLOW = sig
+module type RFLOW = sig
```

Comment:
according to @yallop there's no need to have `RFLOW`, but instead magic runes in implementations of `FLOW` can be used...